app/controllers/github_events_controller.rb

```diff
@@ -28,14 +28,14 @@ def process_pull_request_comment
     comment = Comment.new(event.comment_hash)
     owner = pull_request.github_user
 
-    if comment.commenter.reviews?(pull_request)
-      review = comment.commenter.reviews.find_by(pull_request: pull_request)
-      review.reject if comment.rejected?
-      review.approve if comment.approved?
-    elsif comment.commenter == owner
+    if comment.commenter == owner
```

Comment:
Changed the order because when a comment on a pr comes in it's more important to know if the commenter is the owner of the pr, than if it is a reviewer.